app/src/userpages/components/StreamPage/Show/ConfigureView/index.jsx

```diff
@@ -174,8 +174,8 @@ export class ConfigureView extends Component<Props, State> {
                                                     value={field.name}
                                                     onChange={(e) => this.onFieldNameChange(field.name, e.target.value)}
                                                     disabled={disabled}
-                                                    actions={[
-                                                        <DropdownActions.Item key="delete" onClick={() => this.deleteField(field.name)}>
+                                                    actions={!disabled && [
+                                                        <DropdownActions.Item disabledkey="delete" onClick={() => this.deleteField(field.name)}>
```

Comment:
What does that `disabledkey` do?

Reply:
no idea how that got in there, it should just be `key`, will change